From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Eric Biggers <ebiggers@google.com>
Date: Wed, 12 Aug 2020 13:55:48 -0700
Subject: _____ANNOTATION: Android vendor hooks_____

Signed-off-by: Eric Biggers <ebiggers@google.com>
---
 ANNOTATIONS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ANNOTATIONS b/ANNOTATIONS
--- a/ANNOTATIONS
+++ b/ANNOTATIONS
@@ -1 +1,2 @@
 pending upstream
+Android vendor hooks
